e2e/tests/stake_supplier.feature

```diff
@@ -14,6 +14,8 @@ Feature: Stake Supplier Namespace
 
     Scenario: User can unstake a Supplier
         Given the user has the pocketd binary installed
+        # Reduce the application unbonding period to avoid timeouts and speed up scenarios.
+        And the "supplier" unbonding period param is successfully set to "1" sessions of "2" blocks
```

Comment:
Why not reuse the standard parameter update technique you have rather than creating something custom?

Reply:
Even if it's more lines of code in the test, I believe it'll be clearer and more sustinable.

Reply:
This is using the same parameter update steps internally, it's just reducing the number of step lines in the feature.